Classes/Sitegeist/Monocle/Controller/PreviewController.php

```diff
@@ -108,23 +51,51 @@ public function moduleAction()
      * @param  array $props
      * @return void
      */
-    public function componentAction($prototypeName, $sitePackageKey, $propSet = '__default', array $props = [])
+    public function indexAction($prototypeName, $sitePackageKey, $propSet = '__default', $props = [])
```

Comment:
Please leave this as `componentAction`, it would be breaking otherwise.

Reply:
I agree this is breaky but would like to use this anyway. If we have to raise the version number for that we should do it.